compiler/modules/front/const.bal

```diff
@@ -19,9 +19,10 @@ type SimpleConst t:SingleValue;
 type FoldError ResolveTypeError;
 
 type FoldContext object {
-    function semanticErr(d:Message msg, s:Position pos, error? cause = ()) returns err:Semantic;
+    function semanticErr(d:Message msg, d:Position|d:Range pos, error? cause = ()) returns err:Semantic;
     // Return value of FLOAT_ZERO means shape is FLOAT_ZERO but value (+0 or -0) is unknown
     function lookupConst(string? prefix, string varName, d:Position pos) returns s:FLOAT_ZERO|t:OptSingleValue|FoldError;
+    function qNameRange(d:Position startPos) returns d:Range;
```

Comment:
Are you using this? If not, then can delete here and in a couple of other places.

Reply:
used only in `ConstFoldContext` (among subtypes) removed from others